symplyphysics/laws/quantum_mechanics/schrodinger/time_dependent_via_time_independent_solution.py

```diff
@@ -0,0 +1,88 @@
+from sympy import Eq, sqrt, exp, I
+from sympy.physics.units import hbar
+from symplyphysics import (
+    Quantity,
+    Symbol,
+    Function,
+    validate_input,
+    validate_output,
+    units,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.conditions.quantum_mechanics import (
+    one_dimensional_wave_function_is_normalized as condition,
+)
+
+# Description
+## When the potential energy is independent of time, the solution of the time-dependent Schrödinger
+## equation can be constructed using the solution of the time-independent equation.
+
+# Law: Psi(x, t) = psi(x) * exp(-1 * (i / hbar) * E * t)
+## Psi(x, t) - solution of time-dependent Schrödinger equation
+## psi(x) - solution of time-independent Schrödinger equation
+## x - position, spatial variable
+## t - time
+## i - imaginary unit
+## hbar - reduced Planck constant
+## E - particle energy
+
+# Conditions
+## - The potential energy is a function independent of time.
+## - This law applies for 1-dimensional systems.
+
+time_dependent_wave_function = Function("time_dependent_wave_function", 1 / sqrt(units.length))
+time_independent_wave_function = Function("time_independent_wave_function", 1 / sqrt(units.length))
+particle_energy = Symbol("particle_energy", units.energy, positive=True)
+position = Symbol("position", units.length, real=True)
+time = Symbol("time", units.time, real=True)
+
+law = Eq(
+    time_dependent_wave_function(position, time),
+    time_independent_wave_function(position) * exp(-1 * (I / hbar) * particle_energy * time),
+)
+
+# Show that the normalization condition still holds
+
+_condition = condition.normalization_condition.subs({
+    condition.position: position,
+    condition.time: time,
+})
+
+_tise_condition = _condition.replace(
```

Comment:
For functions `replace` works every time unlike `subs` and it's generally more convenient because I don't need to think about function arguments, `sympy` will replace everything under the hood itself